data_structures/src/data_request.rs

```diff
@@ -370,33 +370,42 @@ pub fn calculate_tally_change(
 pub fn calculate_witness_reward(
     commits_count: usize,
     reveals_count: usize,
-    honests_count: usize,
-    dr_output: &DataRequestOutput,
-    collateral_minimum: u64,
+    liars_count: usize,
+    errors_count: usize, // it is an out of consensus error
```

Comment:
Perhaps we can simplify the API by using something similar to `RequestResult` as an input to `calculate_witness_reward`.